bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/device/MessageHandler.java

```diff
@@ -381,6 +381,29 @@ public void handleMessage(int group, byte cmd1, Msg msg,
 		}
 	}
 
+	public static class RampOnOffSwitchHandler extends MessageHandler {
+		RampOnOffSwitchHandler(DeviceFeature p) { super(p); }
```

Comment:
The same handler is invoked using pseudo-key 0x19 (DIRECT ACK handling in MessageDispatcher) for both 0x34 and 0x35 responses.